addons/website/static/src/xml/website.editor.xml

```diff
@@ -7,7 +7,7 @@
 
     <!-- Custom checkbox (material-design-like toggle) -->
     <t t-name="website.components.switch">
-        <label class="o_switch" t-att-for="id">
+        <label class="o_switch dropdown-item m-0" t-att-data-view-key="key">
```

Comment:
This template is generic, it should not have a data-view-key param. Add it by yourself on the JS side for your specific use case.

Reply:
@qsm-odoo  all the changes are done.

cc @tde-banana-odoo